nusantara/utils/schemas/pairs.py

```diff
@@ -12,3 +13,14 @@ def features(label_names = ["Yes", "No"]):
             "label": datasets.ClassLabel(names=label_names),
         }
     )
+
+
+def features_with_continuous_label():
```

Comment:
Introducing a new schema for pairs with continuous labels. Might be useful for  #137  too. But let me know what you guys think!